src/libs/actions/IOU.js

```diff
@@ -1888,7 +1890,8 @@ function resetMoneyRequestCategory() {
  * @param {Object[]} participants
  */
 function setMoneyRequestParticipants(participants) {
-    Onyx.merge(ONYXKEYS.IOU, {participants});
+    // TODO: temporarily we want to save both participants and participantAccountIDs, then we can remove participants (and rename the function)
```

Comment:
Maybe it's worth to mention the issue here, because "temporarily" could become very long